pandas/io/tests/parser/common.py

```diff
@@ -439,6 +439,13 @@ def test_get_chunk_passed_chunksize(self):
         piece = result.get_chunk()
         self.assertEqual(len(piece), 2)
 
+    def test_read_chunksize_generated_index(self):
+        # GH 12185
```

Comment:
hmm is this already done for the python parser? (e.g. the progressive numbering for chunks)?